packages/amplify/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.0.0-next.0 (2022-08-02)
+
+Initial developer preview release.
+
+### Developer Preview
+
+The Amplify Flutter libraries are being rewritten in Dart. This version is part of our developer preview for all platforms and is **not** intended for production usage. Please follow our [Web](https://github.com/aws-amplify/amplify-flutter/issues/234) and [Desktop](https://github.com/aws-amplify/amplify-flutter/issues/133) support tickets to monitor the status of supported categories. We will be releasing support incrementally to all platforms.
```

Comment:
```suggestion
The Amplify Flutter libraries are being rewritten in Dart. This version is part of our developer preview for all platforms and is **not** intended for production usage. Please follow our [Web](https://github.com/aws-amplify/amplify-flutter/issues/234) and [Desktop](https://github.com/aws-amplify/amplify-flutter/issues/133) support tickets to monitor the status of supported categories. We will be releasing web and desktop support for other Amplify packages incrementally.
```

Reply:
This suggestion - should you choose to accept it - is applicable to all other CHANGELOGs.

Reply:
Good idea. Changed it slightly to
> We will be releasing Web and Desktop support for all Amplify categories incrementally.